README.md

```diff
@@ -203,8 +203,17 @@ Did everything work as expected? Great - now onto the production site.
 Make sure your changes are deployed to the production server (i.e. you've cut a release with your changes).
 
 1. Look at the times cron tasks are run (specified in [`scripts/lametro-crontasks`](https://github.com/datamade/la-metro-councilmatic/blob/master/scripts/lametro-crontasks)), and plan to rebuild the index inbetween those times. Rebuilding the index will take a few minutes, so plan accordingly.
-2. Merge in the pull request and deploy.
+2. Deploy your changes to production.
 3. As above: shell into the server, go to the `lametro` repo, then remove and rebuild the Solr container.
+```
+ssh ubuntu@boardagendas.metro.net
+cd /home/datamade/lametro-production
```

Comment:
Annoyingly....hehe...we call the production repo, simply, `lametro`. This should be:

```
cd /home/datamade/lametro
```

Reply:
Aha! Done.